google/ggrc-core
# GGRC-7334 Get static templates synchronously without using AJAX request

# Issue description

We have complex obsolete logic for getting templates (as example for controllers e.g. "src/ggrc-client/js/controllers/dashboard_controller.js") which can be simplified and optimized.
Proposal:
 1) Remove using of AJAX for getting templates (templates can be accessed from GGRC.Templates. 2) Remove `ajaxTransport` and `ajaxPrefilter` from "src/ggrc-client/js/helpers.js".
3) Remove "/static/templates" prefix when registering templates with can.stache.registerPartial function.

# Steps to test the changes

1. Open any pages, modals.
2. Verify that everything works properly. 

# Solution description

1. Removed `ajaxTransport` to get templates and replace with getting directly from GGRC.Templates
2. Removed prefix `/static/templates` for templates. 
3. Removed GGRC.templates_path.
4. Removed `ajaxPrefilter` for getting templates.

# Sanity checklist




# PR Review checklist
